blog/2024-11-26-camera-deprecations.md

```diff
@@ -0,0 +1,17 @@
+---
+author: Robert Resch
+authorURL: https://github.com/edenhaus
+authorImageURL: https://avatars.githubusercontent.com/u/26537646?s=96&v=4
+title: "Camera API changes"
+---
+
+With Home Assistant `2024.11` we added WebRTC for most cameras. To add support for it we needed to refactor and improve the camera entity.
+Today we would like to announce that with the upcoming Home Assistant release `2024.12` the following methods are deprecated and will be removed with Home Assistant version `2025.6`:
+
+- The property `frontend_stream_type` will be removed. As of `2024.11` Home assistant will identify the frontend stream type by checking if the camera entity implements the native WebRTC methods ([#130932](https://github.com/home-assistant/core/pull/130932)). Card developers can use the new websocket command `camera/capabilities` to get the frontend stream types.
```

Comment:
```suggestion
- The property `frontend_stream_type` will be removed. As of `2024.12` Home assistant will identify the frontend stream type by checking if the camera entity implements the native WebRTC methods ([#130932](https://github.com/home-assistant/core/pull/130932)). Card developers can use the new websocket command `camera/capabilities` to get the frontend stream types.
```
Should this be "As of 2024.12" instead?

Reply:
No 11 is here correct. Since the November release and WebRTC, we have the camera capabilities, and the frontend only uses them instead of the frontend stream type. That's the reason we deprecate it